start-site/src/main/resources/application.yml

```diff
@@ -867,6 +867,8 @@ initializr:
               version: 3.20.6
             - compatibilityRange: "[3.0.0-M1,3.1.0-M1)"
               version: 4.0.0-M3
```

Comment:
Should this change too or is it intentional that Spring Boot 3.0.x users continue to use Camel 4.0.0-M3?

Reply:
Ah yeah those Milestone releases are essentially no longer supported, and ideally we would only have the latest 4.0-XX until we have a GA release.

I can do a new PR